.infrastructure/helm-chart/todoapp/Chart.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v2
+name: todoapp
+description: Todoapp server Helm chart
+type: application
+version: 0.1.0
+appVersion: "1.16.0"
+dependencies:
+- name: mysql
```

Comment:
You need to specify the repository URL for the MySQL dependency. Add a `repository` field under the `mysql` dependency to indicate where Helm can find this chart.